Shared/Sources/ColourSection.swift

```diff
@@ -6,3 +6,150 @@ struct ColourSection {
+    ColourSection(title: "GROUPED CONTENT BACKGROUND", rows: [
+      ColourItem(name: #keyPath(UIColor.systemGroupedBackground), color: .systemGroupedBackground),
+      ColourItem(name: #keyPath(UIColor.secondarySystemGroupedBackground), color: .secondarySystemGroupedBackground),
+      ColourItem(name: #keyPath(UIColor.tertiarySystemGroupedBackground), color: .tertiarySystemGroupedBackground)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var separatorSection: ColourSection {
+    ColourSection(title: "SEPARATOR", rows: [
+      ColourItem(name: #keyPath(UIColor.separator), color: .separator),
+      ColourItem(name: #keyPath(UIColor.opaqueSeparator), color: .opaqueSeparator)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var linkSection: ColourSection {
+    ColourSection(title: "LINK", rows: [
+      ColourItem(name: #keyPath(UIColor.link), color: .link)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var adaptableSection: ColourSection {
+    ColourSection(title: "ADAPTABLE", rows: [
+      ColourItem(name: #keyPath(UIColor.systemBlue), color: .systemBlue),
+      ColourItem(name: #keyPath(UIColor.systemGreen), color: .systemGreen),
+      ColourItem(name: #keyPath(UIColor.systemIndigo), color: .systemIndigo),
+      ColourItem(name: #keyPath(UIColor.systemOrange), color: .systemOrange),
+      ColourItem(name: #keyPath(UIColor.systemPink), color: .systemPink),
+      ColourItem(name: #keyPath(UIColor.systemPurple), color: .systemPurple),
+      ColourItem(name: #keyPath(UIColor.systemRed), color: .systemRed),
+      ColourItem(name: #keyPath(UIColor.systemTeal), color: .systemTeal),
+      ColourItem(name: #keyPath(UIColor.systemYellow), color: .systemYellow)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var adaptableGraySection: ColourSection {
+    ColourSection(title: "ADAPTABLE GRAY", rows: [
+      ColourItem(name: #keyPath(UIColor.systemGray), color: .systemGray),
+      ColourItem(name: #keyPath(UIColor.systemGray2), color: .systemGray2),
+      ColourItem(name: #keyPath(UIColor.systemGray3), color: .systemGray3),
+      ColourItem(name: #keyPath(UIColor.systemGray4), color: .systemGray4),
+      ColourItem(name: #keyPath(UIColor.systemGray5), color: .systemGray5),
+      ColourItem(name: #keyPath(UIColor.systemGray6), color: .systemGray6)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var nonAdaptableSection: ColourSection {
+    ColourSection(title: "NONADAPTABLE", rows: [
+      ColourItem(name: #keyPath(UIColor.darkText), color: .darkText),
+      ColourItem(name: #keyPath(UIColor.lightText), color: .lightText)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var transparentSection: ColourSection {
+    ColourSection(title: "TRANSPARENT", rows: [
+      ColourItem(name: #keyPath(UIColor.clear), color: .clear)
+    ])
+  }
+
+  @available(iOS 13.0, *)
+  private static var fixedSection: ColourSection {
```

Comment:
This individual section doesn't need to be in individual static var as they are not being called anywhere else outside the class. You can simply have the section inside `coloursData` var before the return.